modules/apps/headless/headless-admin-content/headless-admin-content-impl/rest-openapi.yaml

```diff
@@ -1,3 +1,207 @@
+                                type: string
+                            type: object
+                        geo:
+                            description:
+                                A point determined by latitude and longitude.
+                            properties:
+                                latitude:
+                                    description:
+                                        The latitude of a point in space.
+                                    format: double
+                                    type: number
+                                longitude:
+                                    description:
+                                        The longitude of a point in space.
+                                    format: double
+                                    type: number
+                            type: object
+                    type: object
+                dataType:
+                    description:
+                        The field type (e.g., image, text, etc.).
+                    readOnly: true
+                    type: string
+                name:
+                    description:
+                        The field's internal name. This is valid for comparisons and unique in the structured content.
+                    type: string
+        DisplayPageTemplate:
+            description:
+                ""
+            properties:
+                actions:
+                    additionalProperties:
+                        additionalProperties:
+                            type: string
+                        type: object
+                    readOnly: true
+                    type: object
+                availableLanguages:
+                    description:
+                        The list of languages the display page template has a translation for.
+                    items:
+                        description:
+                            The list of languages the display page template has a translation for.
+                        type: string
+                    readOnly: true
+                    type: array
+                creator:
+                    $ref: "#/components/schemas/Creator"
+                customFields:
+                    items:
+                        $ref: "#/components/schemas/CustomField"
+                    type: array
+                dateCreated:
+                    description:
+                        The display page template's creation date.
+                    format: date-time
+                    readOnly: true
+                    type: string
+                dateModified:
+                    description:
+                        The last time any field of the display page template was changed.
+                    format: date-time
+                    readOnly: true
+                    type: string
+                displayPageSettings:
+                    $ref: "#/components/schemas/DisplayPageTemplateSettings"
+                key:
```

Comment:
Don't know if it is better {shemaName} + Id.

Check for this pattern in other use cases